### PR TITLE
graphviz3 manual migration

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
This should be ready for review @conda-forge/topologytoolkit 

We should also re-run #21 once this PR is merged.

**Comment by julien-tierny** [comment]
hi @lhofmann 
it seems that some osx checks failed. do you have any idea where this could come from?

**Comment by lhofmann** [comment]
> hi @lhofmann it seems that some osx checks failed. do you have any idea where this could come from?

This is caused by conda resolving icu to the wrong version on OSX. Not sure what exactly causes this.

A workaround is to add the following to the `host` and `run` sections in recipe/metal.yaml
```
- icu ==86  # [osx and TTK_WITH_PARAVIEW]
```
@Tobias-Fischer

**Comment by Tobias-Fischer** [comment]
Sorry, I didn't see that some builds have failed.

We shouldn't pin `icu` to a particular version here in this recipe - instead, `icu` is pinned globally for conda-forge. I added `icu` to the host deps. For now without the `# [osx and TTK_WITH_PARAVIEW]` selector, but if you prefer, I can add it @lhofmann

**Comment by Tobias-Fischer** [comment]
This needs https://github.com/conda-forge/paraview-feedstock/pull/155 where `icu` is mistakenly not in the dependency list at the moment. Once this PR is merged, we can retrigger the build here.

**Comment by lhofmann** [comment]
@conda-forge-admin, please rerender

**Comment by lhofmann** [comment]
@conda-forge-admin, please restart ci

**Comment by Tobias-Fischer** [comment]
This fails because of `- paraview ==5.9.1=*_qt  # [TTK_WITH_PARAVIEW and not win]`. Any reason to pull in an old version of paraview (which wasn't migrated and thus fails)?

**Comment by Tobias-Fischer** [comment]
So something has changed in paraview 5.10, and topologytoolkit does not yet seem to be compatible with these changes (at least not the latest release, although the main branch seems to be).

Does anyone know when topologytoolkit has planned a new release? The options are to wait for the new release and build the migration on top of that, or do the icu fix for paraview 5.9 so it applies for this PR.

**Comment by julien-tierny** [comment]
I checked and the builds run fine but one of the tests fails (the one using the `pvpython` API).
The problem is that this API changed a little bit from ParaView 5.9 to 5.10 and the following TTK example file is causing an issue: `examples/pvpython/example.py`

This file is faulty in the version 1.0.0 of TTK but this has been fixed on the dev branch of TTK (https://github.com/topology-tool-kit/ttk). Would it be possible to simply:
1) use TTK version 1.0.0
2) replace the file `examples/pvpython/example.py` with the version from TTK's dev branch?

If that's easy to do, could you explain to me how to do it?

In principle that should work (the problem is that we're not expecting another TTK release before October). 

Another (quick) option would be to simply skip that specific test for the time being.
Thanks for letting me know your thoughts.

**Comment by julien-tierny** [comment]
(to skip the test, it looks like commenting line 121 of `recipe/meta.yaml` would do the trick)

**Comment by Tobias-Fischer** [comment]
Seems like this is ready for review/merge @julien-tierny @lhofmann

**Comment by lhofmann** [comment]
LGTM, thanks a lot for your work!

Looks like the Windows ParaView builds are timing out, let's see if we can get them to the testing stage before merging.

@conda-forge-admin, please restart ci

**Comment by Tobias-Fischer** [comment]
I had a look at the win builds just now - while it seems like a timeout at first sight, they actually get stuck with two different errors:

Build 1:
```
FAILED: Library/lib/ttkBarycentricSubdivisionCS.lib 
cmd.exe /C "cd . && %BUILD_PREFIX%\Library\bin\llvm-lib.exe /nologo /machine:x64 /out:Library\lib\ttkBarycentricSubdivisionCS.lib paraview\singlePlugin\CMakeFiles\ttkBarycentricSubdivisionCS.dir\CMakeFiles\ttkBarycentricSubdivisionCS\ttkBarycentricSubdivisionClientServer.cxx.obj paraview\singlePlugin\CMakeFiles\ttkBarycentricSubdivisionCS.dir\CMakeFiles\ttkBarycentricSubdivisionCS\ttkBarycentricSubdivisionModuleClientServer.cxx.obj paraview\singlePlugin\CMakeFiles\ttkBarycentricSubdivisionCS.dir\CMakeFiles\ttkBarycentricSubdivisionCSInit.cxx.obj  && cd ."
Access is denied.
```

Build 2:
```
FAILED: Library/lib/ttkArrayEditorCS.lib 
```


No idea what's going on there.

File: recipe/meta.yaml
Comment (lhofmann) [inline recipe/meta.yaml:24]: Please change to `skip: True  # [TTK_WITH_PARAVIEW and win]` for now, since this times out.

**Comment by lhofmann** [comment]
Might be an artifact of the cancellation, the dashboard says `The job running on agent Azure Pipelines 22 ran longer than the maximum time of 360 minutes.` on https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=525787&view=results

Let's just not build Windows for now.

**Review by lhofmann — APPROVED** [review]

**Comment by Tobias-Fischer** [comment]
Cool, this should be ready now :)

**Pull request merged**
